manager/client/main.go

```diff
@@ -31,8 +32,8 @@ func run(timeout time.Duration, cmd string, args ...string) {
 }
 
 func polling(client rpc_license.LicenseClient, username string, quantity int32, vendor string, feature string,
-	cmd string, args ...string) {
-	timeout := 1800 * time.Second
+	timeout int, cmd string, args ...string) {
```

Comment:
pass timeout as a `time.Duration`, not as an int, and get rid of the conversion below.